src/main/java/com/epam/izh/rd/online/entity/Author.java

```diff
@@ -19,5 +19,73 @@
  * 6) Переопределить метод toString с выводом всех полей (не забывайте alt+inset)
  */
 public class Author {
+    private String name;
```

Comment:
Synthetic Id (e.g. incremental number or UUID) field helps you to distinguish different entities with the same fields.

Reply:
I see, thanks. The task was implement exactly the structure of class. Because there is a check on column name and their existences.